lib/galaxy/util/profiling.py

```diff
@@ -0,0 +1,22 @@
+"""
+Use this context handler to profile slow parts of the code.
+http://docs.sqlalchemy.org/en/latest/faq/performance.html#code-profiling
+"""
+import contextlib
+import cProfile
+import pstats
+import StringIO
+
+
+@contextlib.contextmanager
+def profiled():
```

Comment:
I did use it to figure out what's slow, I'll remove it (and maybe put that on pypi if I can't find something there, it's quite handy).